connectors/src/connectors/github/lib/github_api.ts

```diff
@@ -881,6 +883,7 @@ export async function processRepository({
         sizeBytes: size,
         documentId,
         parentInternalId,
+        /// reversing the parents here since the convention is bottom to top
```

Comment:
I don't see the revert associated with this comment?

Reply:
I removed the reverse but forgot to remove the comment, doing

Reply:
done